frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileType.java

```diff
@@ -32,11 +32,11 @@ public String getDescription() {
 
   @Override
   public String getDefaultExtension() {
-    return "";
+    return "machete";
   }
 
   @Override
-  public @Nullable Icon getIcon() {
-    return null;
+  public Icon getIcon() {
+    return AllIcons.Actions.Menu_cut;
```

Comment:
Can we have git-machete logo as the icon? ;)

Reply:
(Changes from #273 may be necessary)

Reply:
:spurdo-thumbs-up: